cmd/sealctl/cmd/token.go

```diff
@@ -30,7 +30,8 @@ func newTokenCmd() *cobra.Command {
 		Use:   "token",
 		Short: "token generator",
 		Run: func(cmd *cobra.Command, args []string) {
-			t, err := runtime.Generator()
+			//TODO: Whether a token is required to encapsulate an interface?
+			t, err := kubernetes.Generator()
```

Comment:
So the caller should know runtime is kubernetes?

Reply:
I don't need to know. I'll just move the table of contents and adjust it later